luxtronik/discover.py

```diff
@@ -1,4 +1,6 @@
-"""Discovery function for Luxtronik heat pump controllers"""
+"""Discovery function for Luxtronik heat pump controllers."""
+from __future__ import annotations
```

Comment:
Not sure about this. We seem to be inconsistent with the import of `annotations`. We can go for it, but should we do this (in another pull request) for all modules?

Reply:
The linter and so in does not allow to assign different types to the same variable.
An int var can not set to None. It has to be declared : int | None.
But this is only possible since Python 3.10.
For <3.10 the annotion import ist needed.

Reply:
I can understand this, just wondering whether we should apply this to other files as well. But I guess we will eventually add linting via pre-commit, etc. later on to the whole repository anyway (via #77).